must-gather/README.md

```diff
@@ -41,4 +41,37 @@ You will get a dump of:
     ```
 
 In order to get data about other parts of the cluster (not specific to ocs) you should
-run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
\ No newline at end of file
+run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+
+### How to Contribute
+
+#### Contribution Flow
+Developers must follow these steps to make a change:
+1. Fork the `openshift/ocs-operator` repository on GitHub.
+2. Create a branch from the master branch, or from a versioned branch (such
+   as `release-4.2`) if you are proposing a backport.
+3. Make changes.
+4. Ensure your commit messages include sign-off.
+5. Push your changes to a branch in your fork of the repository.
+6. Submit a pull request to the `openshift/ocs-operator` repository.
+7. Work with the community to make any necessary changes through the code
+   review process (effectively repeating steps 3-7 as needed).
+
+#### Commit and Pull Request Messages
+
+- The message must begin with must-gather and a colon followed by single line description.
```

Comment:
add a subsection and give an example such as.
```
must-gather: adding processes to the background
```
The more example you give the better it is for new desvs who want to contribute, will understand.

Reply:
also recommend small case for `must-gather` tag.